stock_outgoing_shipment_report/views/stock_outgoing_shipment_report_views.xml

```diff
@@ -8,12 +8,23 @@
                 <field name="reference_number" />
                 <field name="purchase_order_number" />
                 <field name="ship_date" invisible="1" />
-                <field name="ship_date_edit" widget="date" string="ShipDate" />
+                <field
+                    name="ship_date_edit"
+                    widget="date"
+                    string="ShipDate"
+                    invisible="1"
+                />
                 <field name="cancel_date" invisible="1" />
-                <field name="cancel_date_edit" widget="date" string="CancelDate" />
+                <field
+                    name="cancel_date_edit"
+                    widget="date"
+                    string="CancelDate"
+                    invisible="1"
+                />
                 <field name="ship_carrier" />
+                <field name="ship_billing" invisible="1" />
```

Comment:
A minor thing but it is better not to change the field position unless there is a reason to it.

Reply:
I return to previous order. Thank you.